src/energyplus/ReverseTranslator/ReverseTranslateChillerElectricASHRAE205.cpp

```diff
@@ -0,0 +1,183 @@
+    if (!ambientTemperatureIndicator_) {
+      LOG(Error, "For " << workspaceObject.briefDescription() << ", cannot find required property 'Ambient Temperature Indicator'");
+      return boost::none;
+    }
+
+    openstudio::model::ChillerElectricASHRAE205 modelObject(extfile.get());
+
+    // Name
+    if (boost::optional<std::string> _name = workspaceObject.name()) {
+      modelObject.setName(_name.get());
+    }
+
+    // Performance Interpolation Method: Optional String
+    if (boost::optional<std::string> _performanceInterpolationMethod =
+          workspaceObject.getString(Chiller_Electric_ASHRAE205Fields::PerformanceInterpolationMethod)) {
+      modelObject.setPerformanceInterpolationMethod(_performanceInterpolationMethod.get());
+    }
+
+    // Rated Capacity: Optional Double
+    if (boost::optional<double> _ratedCapacity = workspaceObject.getDouble(Chiller_Electric_ASHRAE205Fields::RatedCapacity)) {
+      modelObject.setRatedCapacity(_ratedCapacity.get());
+    }
+
+    // Sizing Factor: Optional Double
+    if (boost::optional<double> _sizingFactor = workspaceObject.getDouble(Chiller_Electric_ASHRAE205Fields::SizingFactor)) {
+      modelObject.setSizingFactor(_sizingFactor.get());
+    }
+
+    // Ambient Temperature Indicator: Required String
+    if (openstudio::istringEqual("Schedule", ambientTemperatureIndicator_.get())) {
+
+      // Ambient Temperature Schedule Name: Optional Object
+      if (auto wo_sch_ = workspaceObject.getTarget(Chiller_Electric_ASHRAE205Fields::AmbientTemperatureScheduleName)) {
+        if (auto mo_sch_ = translateAndMapWorkspaceObject(wo_sch_.get())) {
+          if (boost::optional<Schedule> sch_ = mo_sch_->optionalCast<Schedule>()) {
+            modelObject.setAmbientTemperatureSchedule(sch_.get());
+          } else {
+            LOG(Warn, workspaceObject.briefDescription() << " has a wrong type for 'Ambient Temperature Schedule Name'");
+          }
+        }
+      } else {
+        LOG(Warn, workspaceObject.briefDescription()
+                    << " has an Ambient Temperature Indicator of type 'Schedule', but no Schedule assigned. Falling back to 'Outdoors'");
+      }
+    } else if (openstudio::istringEqual("Zone", ambientTemperatureIndicator_.get())) {
+
+      // Ambient Temperature Zone Name: Optional Object
+      if (auto wo_z_ = workspaceObject.getTarget(Chiller_Electric_ASHRAE205Fields::AmbientTemperatureZoneName)) {
+        if (auto mo_ = translateAndMapWorkspaceObject(wo_z_.get())) {
+          // Zone is translated, and a Space is returned instead
+          if (boost::optional<Space> space_ = mo_->optionalCast<Space>()) {
+            if (auto z_ = space_->thermalZone()) {
+              modelObject.setAmbientTemperatureZone(z_.get());
+            }
+          } else {
+            LOG(Warn, workspaceObject.briefDescription() << " has a wrong type for 'Ambient Temperature Zone Name'");
+          }
+        }
```

Comment:
Initially I messed up and didn't realize that translateAndMapWorkspaceObject for a Zone returns a Space.